lib/MusicBrainz/Server/Entity/URL/NicoNicoVideo.pm

```diff
@@ -0,0 +1,21 @@
+package MusicBrainz::Server::Entity::URL:NicoNicoVideo;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'Niconico' }
```

Comment:
If you want to make it localizable to Japanese, please use `l` function as follows:
https://github.com/metabrainz/musicbrainz-server/blob/abde1520f200e33f1177a52e488df06cb40d2fb4/lib/MusicBrainz/Server/Entity/URL/IMSLP.pm#L4
https://github.com/metabrainz/musicbrainz-server/blob/abde1520f200e33f1177a52e488df06cb40d2fb4/lib/MusicBrainz/Server/Entity/URL/IMSLP.pm#L31